spec/models/miq_server/rhn_mirror_spec.rb

```diff
@@ -34,7 +34,7 @@
 
       expect(FileUtils).to receive(:mkdir_p).with("/repo/mirror")
       expect(MiqApache::Conf).to receive(:create_conf_file).once.and_return(true)
-      expect(FileUtils).to receive(:rm).with("/etc/httpd/conf.d/manageiq-https-mirror.conf", :force => true)
+      expect(FileUtils).to receive(:rm).with("/etc/httpd/conf.d/manageiq-https-mirror.conf", hash_including(:force => true))
```

Comment:
See commit, it's basically changed from an options hash to kwargs so the expectations on arguments is different now.